### ISSUE TITLE
Net.DDP.Client missing Newtonsoft.Json

### ISSUE THREAD

**Issue by gb96** [sonyarouje/DDPClient.NET#6]
Clean checkout had build errors.  Could not find Newtonsoft.Json.

To fix I had to open NuGet package manager and tick Json.Net for the project Net.DDP.Client

I am running Visual Studio 2013.

**Comment by sonyarouje** [comment]
Hi GB,
   Newtonsoft Json is not checkedin as part of this project. It should be downloaded from nuget.

Regards,
Sony

**Comment by gb96** [comment]
Yes I know but your nuget config is broken... like I told you :)

**Comment by sonyarouje** [comment]
I am sorry Greg, misunderstood your comment. I will fix that.

**Comment by gb96** [comment]
Thanks Sony.  Actually if I don't checkout your solution and just use your published NuGet package DDPClient.Net version 2.3.21.1 then this doesn't matter, it all just works :)  But some people might still want to checkout and build DDPClient themselves.

P.S. thanks for this implementation, it seems to work very well!

**Comment by gb96** [comment]
P.S. the NuGet package is published via https://github.com/OzTK/DDPClient.NET and current published version 0.4.1 is a little behind.  For some reason it is throwing System.InvalidOperationException.  I will have to go back to using your code instead of NuGet for now...

**Comment by sonyarouje** [comment]
Thanks Greg. I am in the process of setting up the dev environment in my new laptop. I will work on it once I set it up.

I am so happy to hear that every thing is working fine. Also it encourage me to start work on this project again. It was dead for so long because I assumed no one was using it.
